pages/plugins.rst

```diff
@@ -76,13 +76,11 @@ which will create a complete plugin project infrastructure with all required cla
 
 Maven is a widely used build tool for Java, that comes pre-installed on many operating systems or can be installed using most package managers. Make sure that you have at least version 3 before you go on.
 
-Use it like this::
+First you should install the latest version of the `graylog-project-cli <https://github.com/graylog2/graylog-project-cli/releases>`_ on your workstation. Use this to bootstrap the meta project in your working directory with the following command::
 
-  $ git clone git@github.com:Graylog2/graylog-project.git
+  $ graylog-project bootstrap github://Graylog2/graylog-project.git
 
-
-This will create a checkout of the meta project in your current work dir. Now change to the ``graylog-project`` directory and follow the ``Install CLI Tool`` and ``Bootstrap`` from its README.
-Bootstrapping the ``garylog-project`` will create the directory ``graylog-project-repos``. It contains all required repositories and will also be the home of your new plugin.
+This will create a checkout of the meta project in your current directory. Now have the ``graylog-project`` and ``graylog-project-repos``. Those two directories contain the meta data and all repositories that are required. The directory ``graylog-project-repos`` will also be the home of your new plugin.
```

Comment:
```suggestion
This will create a checkout of the meta project in your current directory. You'll see both ``graylog-project`` and ``graylog-project-repos``. Those two directories contain the meta data and all repositories that are required. The directory ``graylog-project-repos`` will also be the home of your new plugin.
```